mandatory/1-syntax-errors.js

```diff
@@ -1,17 +1,27 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
+let x = 3;
+let y = 4;
+let z = 6;
+function addNumbers(x, y, z) {
+  return x + y + z;
+}
+console.log(addNumbers);
+
```

Comment:
```suggestion
function addNumbers(a, b, c) {
  return a + b + c;
}
```

Hey, have a look here. We already have parameters inside the function. The task is just to correct syntax